### PR TITLE
Colors are now Relativ, And Animation of Down Scroll Button

### PR DESCRIPTION
**Pull request by Michiocre**
Nice

### CODE REVIEW

**Comment by Michiocre** [comment]
Now also with Title and Logo

**Comment by Michiocre** [comment]
Has been cleaned up

**Comment by Michiocre** [comment]
All the Problems are now solved, i think

File: spoti-vote/public/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/spotiLogo.png">
```
Comment (Gabsii) [inline spoti-vote/public/index.html]: convert to .ICO file pls

File: spoti-vote/src/components/Buttons/ScrollDown.jsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 import FontAwesomeIcon from '@fortawesome/react-fontawesome';
 import {faAngleDown} from '@fortawesome/fontawesome-free-solid';
+let color = require('../../css/colors.js');
 
 let defaultStyle = {
+	position: 'absolute',
+	bottom: '0',
 	marginLeft: 'auto',
 	marginRight: 'auto',
 	marginTop: 'auto',
```
Comment (Gabsii) [inline spoti-vote/src/components/Buttons/ScrollDown.jsx]: remove marginTop. unnecessary afaic

File: spoti-vote/src/components/Header.jsx [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import React, {Component} from 'react';
-import logo from '../img/sbahn.png'
+import logo from '../img/spotiLogo.png';
```
Comment (Gabsii) [inline spoti-vote/src/components/Header.jsx]: preferably a .SVG File

Comment (Michiocre) [reply spoti-vote/src/components/Header.jsx]: The svg file has problems, it uses an S from a diffrent font than the one i designed it with, ill try to solve this problem tho

**Comment by Michiocre** [comment]
All issues are now gone

**Pull request merged**